src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2757,6 +2757,9 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
                 loadDataFiles(problem, dataFiles, answerFiles, sampleDataDirectory, loadExternalFile);
             }
 
+            
```

Comment:
Is there any particular reason we can't do the refactor now since it should be pretty simple to do?